trunk/src/mpi/zoltan_lb_param.F90

```diff
@@ -270,10 +270,19 @@ subroutine zoltan_lb_param_graph(Ierr_out)
    call zoltan_w_handle_err(ierr,'Zoltan_Set_Param')
    if (ierr > Ierr_out) Ierr_out = ierr
 !
-!..PARMETIS_OUTPUT_LEVEL
-!     0: no output
-!     1: print timing info
-   ierr = Zoltan_Set_Param(zz,'PARMETIS_OUTPUT_LEVEL','0')
+   ierr = Zoltan_Set_Param(zz,'PARMETIS_ITR','1')
+   call zoltan_w_handle_err(ierr,'Zoltan_Set_Param')
+   if (ierr > Ierr_out) Ierr_out = ierr
+!
+   ierr = Zoltan_Set_Param(zz,'CHECK_GRAPH','1')
+   call zoltan_w_handle_err(ierr,'Zoltan_Set_Param')
+   if (ierr > Ierr_out) Ierr_out = ierr
+!
+   ierr = Zoltan_Set_Param(zz,'SCATTER_GRAPH','2')
+   call zoltan_w_handle_err(ierr,'Zoltan_Set_Param')
+   if (ierr > Ierr_out) Ierr_out = ierr
+!
+   ierr = Zoltan_Set_Param(zz,'IMBALANCE_TOL','1.02')
```

Comment:
Can you add a line describing what these do. And, does the `PARMETIS_OUTPUT_LEVEL` not work?

Reply:
Will do! The `PARMETIS_OUTPUT_LEVEL` is 0 by default I think; I set it in my MG solver and must have forgotten to put it back here. It definitely works, I can revert that deletion.

Reply:
Thanks!